src/openforms/registrations/contrib/objects_api/models.py

```diff
@@ -0,0 +1,54 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from solo.models import SingletonModel
+from zgw_consumers.constants import APITypes
+
+
+class ObjectsAPIConfig(SingletonModel):
+    """
+    global configuration and defaults
+    """
+
+    objects_service = models.OneToOneField(
+        "zgw_consumers.Service",
+        verbose_name=_("Objects API"),
+        on_delete=models.PROTECT,
+        limit_choices_to={"api_type": APITypes.orc},
+        related_name="objects_objects_config",
```

Comment:
`related_name` does not make any sense with `SingletonModel` since there's only ever one record in the database.

```suggestion
        related_name="+",
```